drivers/src/memory_layout.rs

```diff
@@ -74,7 +75,8 @@ pub const DPE_SIZE: u32 = 5 * 1024;
 pub const PCR_RESET_COUNTER_SIZE: u32 = 1024;
 pub const AUTH_MAN_IMAGE_METADATA_MAX_SIZE: u32 = 7 * 1024;
 pub const IDEVID_CSR_SIZE: u32 = 1024;
-pub const DATA_SIZE: u32 = 27 * 1024;
+pub const FMC_ALIAS_CSR_SIZE: u32 = 1024;
```

Comment:
Did you confirm this was sufficient given the FMC ALIAS CSR is going to have some extra fields over IDEV?